pydantic_ai_slim/pydantic_ai/_agent_graph.py

```diff
@@ -301,16 +301,15 @@ async def _stream(
             ctx.state.message_history, model_settings, model_request_parameters
         ) as streamed_response:
             self._did_stream = True
-            ctx.state.usage.incr(_usage.Usage(), requests=1)
+            ctx.state.usage.requests += 1
```

Comment:
Was this change on purpose? I believe the intent of using a method call here was to ensure it raised an error if the requests count went above the allowed number, but maybe that's done elsewhere?

Reply:
Well whatever the original intent was, I can see it's not doing that now. So this makes sense.